legend-engine-language-pure-modelManager-sdlc/src/main/java/org/finos/legend/engine/language/pure/modelManager/sdlc/SDLCLoader.java

```diff
@@ -180,7 +180,9 @@ else if (context.sdlcInfo instanceof AlloySDLC)
                     }
                     else
                     {
-                        throw new RuntimeException("The following PackageableElementPointers:" + missingPaths.toString() + " do not exist in the project data loaded from the metadata server");
+                        throw new RuntimeException("The following PackageableElementPointers:" + missingPaths + " do not exist in the project data loaded from the metadata server. " +
```

Comment:
Minor question: this is an error message for end users (e.g. in Studio) right? Does it make sense to say PackageableElementPointers to end users? Will they understand what that means?

Reply:
Shall i just change it to 'entity' ?

Reply:
That's a good suggestion: I think entity (or entities) makes more sense than PackageableElementPointers